Makefile

```diff
@@ -325,14 +319,9 @@ generate-uml: $(patsubst %.uml,%.png,$(wildcard site/img/uml/*.uml))
 %.png: %.uml
 	cd `dirname $@` && plantuml `basename "$^"`
 
-.PHONY: site-devel
-site-devel: ## Launch the website in a Docker container
```

Comment:
do we have a replacement for this for testing changes to the site locally?

Reply:
From the site directory, you can run `hugo serve` which will do that. We can keep the make task or use a docker image if desired. We had the docker image before since ruby was difficult to get running properly across systems.

Reply:
oh sweet, might be cool to keep the make target and have it run the hugo command, just so we don't have to remember it and can updating it with any args we need to customize the command

maybe good to keep for later with an issue while we try to get things working

Reply:
Yeah I'd say let's keep the make target and just have it run `hugo serve`, we can require that as a local dependency for now (`brew install hugo` on mac). That way it's documented how to do.

Reply:
I added back the `hugo serve` as `make site-devel`. Would like to expand further with a docker image if the binary is not locally available, also helps speed up folks. Right now the community hugo docker image can't be pulled from dockerhub, not sure if I'm rate limited, but can follow up.